adapters/Pangle/Adapter/Adapter.m

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 Google LLC
```

Comment:
Ditto, this file can actually be removed, as well as the parent `Adapter` directory.